bin/templates/cordova/lib/PackageJsonParser.js

```diff
@@ -29,13 +30,28 @@ class PackageJsonParser {
         };
     }
 
-    configure (config) {
+    configure (config, projectPackageJson) {
         if (config) {
             this.package.name = config.packageName() || 'io.cordova.hellocordova';
             this.package.displayName = config.name() || 'HelloCordova';
             this.package.version = config.version() || '1.0.0';
             this.package.description = config.description() || 'A sample Apache Cordova application that responds to the deviceready event.';
 
+            const cordovaDependencies = Object.keys(projectPackageJson.dependencies)
+                .filter((npmPackage) => /^cordova(?!-plugin)-/.test(npmPackage));
+
+            // If Cordova dependencies are detected in "dependencies" of "package.json" warn for potential app package bloating
+            if (cordovaDependencies.length) {
+                events.emit('warn', `The following Cordova package(s) were detected as "dependencies" in the projects "package.json" file.
+\t- ${cordovaDependencies.join('\n\t- ')}
+
+It is recommended that all Cordova packages are defined as "devDependencies" and is safe to be moved manually.
```

Comment:
grammar sounds broken here

Reply:
@janpio  How about this?

```suggestion
It is recommended that all Cordova packages are defined as "devDependencies" in the "package.json" file. It is safe to move them manually.
```

Reply:
Yep.

Reply:
Current code again shows state without this change.

Reply:
Sorry, it seems I added to batch but not commit. ha.